Userland/Libraries/LibWeb/HTML/HTMLObjectElement.cpp

```diff
@@ -215,7 +215,11 @@ void HTMLObjectElement::run_object_representation_handler_steps(Optional<String>
     // * If the resource type starts with "image/", and support for images has not been disabled
     // FIXME: Handle disabling image support.
     else if (resource_type.has_value() && resource_type->starts_with("image/"sv)) {
-        // FIXME: If the object element's nested browsing context is non-null, then it must be discarded and then set to null.
+        // If the object element's nested browsing context is non-null, then it must be discarded and then set to null.
```

Comment:
Totally optional if you want to include it in this commit, but we need to do the same in `run_object_representation_fallback_steps`

Reply:
You already did, 4 hours ago :D
https://github.com/alimpfard/serenity/commit/d744f04a03aa15e1c55316b1626efdf84f45375f

Reply:
:man_facepalming: too many object branches, I thought I only had that locally haha